### PR TITLE
Adds magillitis, a traitor serum for geneticists & CMOs

### PR DESCRIPTION
**Pull request by ShizCalev**
It is available for purchase at the cost of 15 telecrystals, and comes in a single-use autoinjector. Upon injection of the serum into a monkey, the monkey will transform into a gorilla.

@The-clown-known-as-Pepe Ask and ye shall receive.

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm [hunk 1]
```diff
@@ -376,6 +376,19 @@
 	M.radiation += 4
 	..()
 
+/datum/reagent/toxin/magillitis
```
Comment (RandomMarine) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: What's the point of this being under the toxin path?

Comment (ShizCalev) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: Just to make it clear that it's a bad chemical. Can move it anywhere if requested.

Comment (RandomMarine) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: It doesn't really matter much. But being under the toxin path just makes it count for liver shenanigans and trekchem anti-toxin. Which seems pointless when it just makes monkeys into gorillas.

**Comment by RandomMarine** [comment]
As another note, it's a simple matter to split the 5u you get in each bottle to 0.1u doses.

Enough to mutate 50 gorillas.

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm (hunk 1 above)
Comment (Cyberboss) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: could probably just do tox damage to non-monkeys

**Comment by Armhulen** [comment]
but gorillas are incredibly easy to get. Make it turn humans into gorillas, and you'd have something to write home about.

**Comment by DoctoreMad** [comment]
I'll be surprised if Gorillas aren't unmerged by a weeks time.
Pre-emptive PR, its totally okay.

**Comment by ShizCalev** [comment]
@Armhulen Easy change, but I don't want to go balls to the wall too early since they're still being tweaked.

**Comment by DoctoreMad** [comment]
I liked experimental syrup more.

**Comment by pubby** [comment]
If the radiation transformation doesn't get removed, traitors can use a microlaser to make gorillas.

**Comment by FantasticFwoosh** [comment]
you want to know the best part about this?

- Its a reagent so you can get it from strange plants in hydroponics

**Comment by The-clown-known-as-Pepe** [comment]
@FantasticFwoosh its like hitting a big rng loterry, you have 30% chance to get a chem in strange plant and then you have to roll this exact chem.

**Comment by FantasticFwoosh** [comment]
As my previous PR proves at #29956, there are ways to play the lottery more than once if the right tools are put in place, and the lottery is always fun game to play.

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 2]
```diff
@@ -1573,6 +1573,19 @@
...
+	name = "Magillitis"
+	id = "magillitis"
+	description = "An experimental serum which causes rapid muscular growth in basic primates. Side-affects may include hypertrichosis, violent outbursts, and an unending affinity for bananas."
+	reagent_state = LIQUID
+	color = "#00f041"
+
+/datum/reagent/magillitis/on_mob_life(mob/living/carbon/M)
+	if(ismonkey(M))
+		M.reagents.del_reagent("magillitis")
+		M.gorillize()
```
Comment (Cyberboss) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: Make it use all 5u first (cycle var i think?)

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 3]
```diff
@@ -65,6 +65,25 @@
 	volume = 100
 	list_reagents = list("nanites" = 80, "synaptizine" = 20)
 
+/obj/item/reagent_containers/hypospray/magillitis
+	name = "experimental autoinjector"
+	desc = "A modified air-needle autoinjector with a small single-use reservoir. It contains an experimental serum."
+	icon_state = "combat_hypo"
+	volume = 5
+	container_type = 0
```
Comment (Cyberboss) [inline code/modules/reagents/reagent_containers/hypospray.dm]: NONE

File: code/modules/reagents/reagent_containers/hypospray.dm [hunk 4]
```diff
@@ -65,6 +65,25 @@
...
+	list_reagents = list("magillitis" = 5)
+
+/obj/item/reagent_containers/hypospray/magillitis/attack(mob/living/M, mob/user)
+	if(!reagents.total_volume)
+		return ..()
+	if(!ismonkey(M))
+		if(user == M)
+			to_chat(user, "<span class='notice'>That's probably not a good idea...</span>")
+		else
+			to_chat(user, "<span class='notice'>I probably shouldn't use that on [M.p_them()]...</span>")
```
Comment (Cyberboss) [inline code/modules/reagents/reagent_containers/hypospray.dm]: Non traitors know this how?

File: code/modules/uplink/uplink_item.dm [hunk 5]
```diff
@@ -1314,6 +1314,13 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 	cost = 14
 	restricted_roles = list("Geneticist", "Chief Medical Officer")
 
+/datum/uplink_item/role_restricted/magillitis_serum
+	name = "Magillitis Serum Autoinjector"
+	desc = "A single-use autoinjector which contains an experimental serum that causes rapid muscular growth in basic primates."
+	item = new /obj/item/reagent_containers/hypospray/magillitis
```
Comment (Cyberboss) [inline code/modules/uplink/uplink_item.dm]: This is supposed to be a path, not an instance

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/uplink/uplink_item.dm (hunk 5 above)
Comment (ShizCalev) [inline code/modules/uplink/uplink_item.dm]: Whoop.

**Comment by RicardoFort** [comment]
Rename it Injector de Maccaco and call it a day

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 6]
```diff
@@ -1573,6 +1573,19 @@
...
+	id = "magillitis"
+	description = "An experimental serum which causes rapid muscular growth in basic primates. Side-affects may include hypertrichosis, violent outbursts, and an unending affinity for bananas."
+	reagent_state = LIQUID
+	color = "#00f041"
+
+/datum/reagent/magillitis/on_mob_life(mob/living/carbon/M)
+	if(ismonkey(M) && volume >= 5)
+		M.reagents.del_reagent("magillitis")
+		return M.gorillize()
+	..()
```
Comment (Cyberboss) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: Call ..() first

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 7]
```diff
@@ -1573,6 +1573,19 @@
...
 
+/datum/reagent/magillitis
+	name = "Magillitis"
+	id = "magillitis"
+	description = "An experimental serum which causes rapid muscular growth in basic primates. Side-affects may include hypertrichosis, violent outbursts, and an unending affinity for bananas."
+	reagent_state = LIQUID
+	color = "#00f041"
+
+/datum/reagent/magillitis/on_mob_life(mob/living/carbon/M)
+	if(ismonkey(M) && volume >= 5)
```
Comment (Cyberboss) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: Change `volume >= 5` to `current_cycle >= 10`

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 8]
```diff
@@ -1573,6 +1573,19 @@
...
+		M.reagents.del_reagent("magillitis")
```
Comment (Cyberboss) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: Remove this line

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Review by Cyberboss — APPROVED** [review]
For 15TC, sure why not

**Pull request merged**